sdk/golang/ccnp/quote/quote.go

```diff
@@ -0,0 +1,191 @@
+/*
+* Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+* SPDX-License-Identifier: Apache-2.0
+ */
+
+package quote
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/binary"
+	"log"
+	"strings"
+	"time"
+
+	pb "github.com/intel/confidential-cloud-native-primitives/sdk/golang/ccnp/quote/proto"
+	pkgerrors "github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+const (
+	UDS_PATH = "unix:/run/ccnp/uds/quote-server.sock"
+	TYPE_TDX = "TDX"
+	TYPE_TPM = "TPM"
+)
+
+type TPMQuote struct{}
+
+type TDXQuote struct {
+	Quote           []uint8    // full TD quote
```

Comment:
So what actually this type TDXQuote is? It looks like a mixture of raw quote, raw report and sub attributes within the quote and reports...

Reply:
This is in sync with python SDK and I agree with this design that keeps both raw quote and parsed items for SDK client's further usage.